### PR TITLE
Add eslint, prettier & github actions

### PR DESCRIPTION
**Pull request by VNDRN**
I've added a basic eslint & prettier config. 
I've also added a github action that checks these + typecheck + runs the tests.

Screenshot 2024-05-07 at 16 11 11

Let me know if the rules need to be adjusted

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "CLI tool for syncing assets from and to Loco",
   "license": "MIT",
   "repository": "robrechtme/loco-cli",
+  "type": "module",
```
Comment (robrechtme) [inline package.json:7]: Why is this needed?

**Review by robrechtme — COMMENTED** [review]
Thank you very much for the contribution 🙏

File: package.json (hunk 1 above)
Comment (VNDRN) [reply package.json]: I've reverted it and fixed an issue I had in another, less intrusive way

**Comment by robrechtme** [comment]
Closing this one in favour of #35
